go/runtime/registry/config.go

```diff
@@ -254,6 +267,9 @@ func newConfig(dataDir string, consensus consensus.Backend, ias ias.Endpoint) (*
 				if !forceNoSGX {
 					break
 				}
+				if runtimeEnv == CfgRuntimeEnvironment {
```

Comment:
This seems wrong?

Reply:
my autocomplete sucks, but the Go type system sucks more

Reply:
That's the reason why the mode is typed :)